gridgain/gridgain
# NBL-4: Node.js thin client: affinity awareness feature

What was done:
- Implementation of the Affinity Awareness feature
- Tests for this feature
- A separate test configuration

TODO:
- Check that jsdoc generates with no issues
- Make an updated instruction for tests (?)
